### DISCUSSION TITLE
Build Summary Reporting for Gatsby Cloud

### DISCUSSION THREAD

**Discussion by AnneFonicelloUW** [gatsbyjs/gatsby#32518] [Ideas / Feature Requests]
My team is looking at how we can build reports on build statistics for our websites in Gatsby Cloud. Currently I'm having to manually fill out a spreadsheet with Build times and Success/Fail rates from build history lists so that we can compile and analyze the data over several days. It would be great if this was something we could just export that information from the GatsbyCloud dashboard, or obtain automatically through another means. Is this something that could be turned into a new feature? Has anyone else tried to do something similar with their websites?

**Comment by LekoArts** [comment]
Hi!

For Gatsby Cloud suggestions please use https://portal.gatsbyjs.com/gatsby-cloud

Thanks!

**Reply by AnneFonicelloUW** [reply]
Ok, thank you!
